api/Worker.json

```diff
@@ -171,10 +171,10 @@
                 "version_added": "57"
               },
               "safari": {
-                "version_added": "14.1"
+                "version_added": "15"
```

Comment:
Seems like I didn't submit my comment here. Is the original 14.1 wrong here, where did that data come from?

Reply:
Yes. It is wrong. I don't know where it came from.

Reply:
Thanks for confirming, no need to track it down if you're confident it's wrong.

Reply:
I verified it was wrong on Safari 14.1 using wpt.live tests. The change appears to have come from this: https://github.com/mdn/browser-compat-data/pull/10749 That PR cites a blogger discussing it, but the blogger discussed it in the context of Safari Technology Preview, not the Safari 14.1 release.

Reply:
Oh, I merged that, oops! @tomayac FYI that this turned out to be wrong.

Reply:
Oops, sorry. I was under the impression the TP version numbers could be used for this. Honest mistake.

Reply:
Ah, and I didn't realize it was intended to mean STP :)

Soon we will have "preview" as an allowed string for things that are only in STP (and Chrome Canary or Firefox Nightly).